web/assets/css/dashboard.less

```diff
@@ -62,9 +62,8 @@
     background: @grey80;
     z-index: 2;
     opacity: 0.9;
-    padding-top: (35px * @scale);
-    padding-bottom: (35px * @scale);
-    padding-left: ((50px + @name-padding) * @scale);
+    padding-top: (64px * @scale);
+    padding-left: ((39px + @name-padding) * @scale);
```

Comment:
Can we get rid of `@name-padding` altogether and just use a fixed `64px` here? `@name-padding` isn't used anywhere else.

Also, looks like the other variables here (`@banner-height`, `@footer-padding`) are also completely unused, so we can remove them too